### ISSUE TITLE
Export FeaturceCollection as a csv file in R?

### ISSUE THREAD

**Issue by helenyang24** [r-spatial/rgee#78]
I'm converting a GEE script to R, and my ultimate goal is to export the data in a csv file. I'm wondering if there is any way to make it doable in R?

**Comment by csaybar** [comment]
Hi maybe this help: https://csaybar.github.io/rgee-examples/
https://github.com/r-spatial/rgee/blob/examples/rgee/03_fc_download.R

**Comment by helenyang24** [comment]
Thanks. When I run the sample code in R, it shows:
```
> library(rgee)
> ee_Initialize(drive = TRUE)
── rgee 1.0.0 ───────────────────────────────────────────────────────────────────────────────────── earthengine-api 0.1.227 ── 
 ✓ email: not_defined
 ✓ Google Drive credentials:
1: luyang21@colby.edu

Selection: ee_user_info()
Enter an item from the menu, or 0 to exit
Selection: 
Enter an item from the menu, or 0 to exit
Selection: # Region of interest
Enter an item from the menu, or 0 to exit
Selection: roi <- ee$Geometry$Point(c(-122.2575, 37.8795)) %>%
Enter an item from the menu, or 0 to exit
Selection:   ee$Geometry$buffer(10000)
Enter an item from the menu, or 0 to exit
Selection: 
Enter an item from the menu, or 0 to exit
Selection: ## 1. Download a small FeatureCollections
Enter an item from the menu, or 0 to exit
Selection: blocks <- ee$FeatureCollection("TIGER/2010/Blocks")
Enter an item from the menu, or 0 to exit
Selection: subset <- blocks$filterBounds(roi)
Enter an item from the menu, or 0 to exit
Selection: sf_subset <- ee_as_sf(x = subset, maxFeatures = 10000)
Enter an item from the menu, or 0 to exit
Selection: plot(sf_subset["countyfp10"])
Enter an item from the menu, or 0 to exit
Selection: 
Enter an item from the menu, or 0 to exit
Selection: 
Enter an item from the menu, or 0 to exit
Selection: ## 2. Download a Large FeatureCollections
Enter an item from the menu, or 0 to exit
Selection: region <- ee$Geometry$Rectangle(-119.224, 34.669, -99.536, 50.064)
Enter an item from the menu, or 0 to exit
Selection: # ee_help(ee$FeatureCollection$randomPoints)
Enter an item from the menu, or 0 to exit
Selection: ee_randomPoints <- ee$FeatureCollection$randomPoints(region, 50000)
Enter an item from the menu, or 0 to exit
Selection: sf_random <- ee_as_sf(x = ee_randomPoints, via = "drive")
Enter an item from the menu, or 0 to exit
Selection: 
```
What does that ''Selection:" mean and what should I enter?

**Comment by csaybar** [comment]
It seems you have multiple Google Drive credentials, you need to select one of them before continue.

**Comment by helenyang24** [comment]
I tried entering my email but the 'Select:' still popped up:
```
Enter an item from the menu, or 0 to exit
Selection: my@email.com

1: my@email.com

Selection: my@email.com

1: my@email.com

Selection: 
```

**Comment by csaybar** [comment]
You must enter 1

**Comment by helenyang24** [comment]
It seems like I still need to select something after entering 1:
```
Selection: library(rgee)
Enter an item from the menu, or 0 to exit
Selection: ee_Initialize(drive = TRUE)
Enter an item from the menu, or 0 to exit
Selection: ee_user_info()
Enter an item from the menu, or 0 to exit
Selection: my@email.com

1:my@email.com

Selection: 1

1: my@email.com

Selection: 1: my@email.com

1: my@email.com

Selection: 
```

**Comment by csaybar** [comment]
It was fixed in rgee v1.0.4
